samples/property/provider/src/provider_impl.rs

```diff
@@ -28,22 +28,22 @@ impl Provider for ProviderImpl {
         request: Request<SubscribeRequest>,
     ) -> Result<Response<SubscribeResponse>, Status> {
         let request_inner = request.into_inner();
-        let id: String = request_inner.id.clone();
-        let uri: String = request_inner.uri;
+        let entity_id: String = request_inner.entity_id.clone();
+        let consumer_uri: String = request_inner.consumer_uri;
 
-        info!("Received a subscribe request from URI {} for id {}", &uri, &id);
+        info!("Received a subscribe request from URI {} for id {}", &consumer_uri, &entity_id);
 
         let mut lock: MutexGuard<HashMap<String, HashSet<String>>> =
             self.subscription_map.lock().unwrap();
-        let get_result = lock.get(&id);
+        let get_result = lock.get(&entity_id);
         if let Some(get_value) = get_result {
```

Comment:
We can simplify this if statement with a match, see my comment on the command provider for the structure